### PR TITLE
Added vanny96's project!

### PR DESCRIPTION
**Pull request by vanny96**
This is a PR template. If you are adding a solution link to the curriculum, leave this as is. If not, delete it and write the message you wish.

Thank you,

The Odin Project team.

### CODE REVIEW

File: web_development_101/the_front_end/project_html_css.md [hunk 1]
```diff
@@ -1981,6 +1981,7 @@ Submit a pull request with a link to your solution in this [file](https://github
 * [João Fonseca's Solution (Easy)](https://github.com/FonsecaJoao/google-homepage) - [View in Browser](https://fonsecajoao.github.io/google-homepage/)
 * [Kenny Chung's Solution (Easy)](https://github.com/kpchungdev/google_home_page) - [View in Browser](https://kpchungdev.github.io/google_home_page/)
 * [Kenny Chung's Solution (Difficult)](https://github.com/kpchungdev/google_search_results_page) - [View in Browser](https://kpchungdev.github.io/google_search_results_page/)
+* [vanny96's Solution (Easy)](https://github.com/vanny96/google-homepage) - [View in Browser](https://vanny96.github.io/google-homepage/) cacca
```
Comment (CouchofTomato) [inline web_development_101/the_front_end/project_html_css.md]: Did you mean to write cacca here?

**Comment by vanny96** [comment]
No I didn't! :O it means "shit" in italian! Now I don't know if I
unconsciously described my feeling toward the results of my project but I
didn't put it there with purpose ahaha you can delete it!

Il ven 1 mar 2019, 21:37 Austin <notifications@github.com> ha scritto:

File: web_development_101/the_front_end/project_html_css.md (hunk 1 above)
Comment (vanny96) [reply web_development_101/the_front_end/project_html_css.md]: No I didn't meant to :/ you can delete it from the code! (cacca means "shit" in italian... I may have used it to test something and it slipped... I'm sorry :/)

**Comment by 105ron** [comment]
Closed, I have added it to the site with https://github.com/TheOdinProject/curriculum/pull/9867 because of the merge conflicts and removed cacca.

**Pull request closed without merging**